HW_06_Task1/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace HW_06_Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("write poem separating the string symbol of \";\"");
+            string poem = Convert.ToString(Console.ReadLine());

```

Comment:
you do not need to use Convert.ToString() because of Console.Readline already returns string type